edx_analytics_transformers/routers/requests_router.py

```diff
@@ -0,0 +1,143 @@
+"""
+Generic router to send events to hosts.
+"""
+import logging
+
+from eventtracking.processors.exceptions import EventEmissionExit
+
+from edx_analytics_transformers.utils.http_client import HttpClient
+from edx_analytics_transformers.django.models import RouterConfigurations
+
+
+logger = logging.getLogger(__name__)
+
+
+class RequestsRouter:
```

Comment:
This router will be enabled through the `EVENT_TRACKING_BACKENDS` configurations.

```
CALIPER_SETTINGS = {
    'ENGINE': 'edx_analytics_transformers.backends.caliper.CaliperBackend',
    'OPTIONS': {
        'routers':{
            'caliper_router': {
                'ENGINE': 'edx_analytics_transformers.routers.requests_router.RequestsRouter',
                'OPTIONS': {
                    'processors':[
                        {
                            'ENGINE': 'edx_analytics_transformers.processors.caliper_envelop.CaliperEnvelopProcessor',
                            'OPTIONS': {
                                'sensor_id': 'http://test.com/sensors'
                            }
                        }
                    ],
                    'backend_name': 'caliper',
                }
            }
        }

    }
}
```

 I've added the configuration in PR description too.

I am also thinking about renaming the router to `FilteredEventsRouter` since the routing strategy varies upon config now. Any thoughts or better suggestions?